+eui/Contents.m

```diff
@@ -0,0 +1,32 @@
+% +EUI Experimental UI package
```

Comment:
is there a reason you chose the name `Contents` ? If we have these descriptive files as README.md, they'll be displayed as readmes in github when navigating to the folder containing the readme. 

https://github.com/cortex-lab/Rigbox/issues/156